app/views/donuts/show.html.erb

```diff
@@ -1,24 +1,48 @@
-<h1><%= @donut.name %></h1>
+<div class="uploaded-donut-content">
+  <h1 class="donut-title"><%= @donut.name %></h1>
+  <%= image_tag(@donut.image, id: "individual-donut", class: "donut-image")%>
+  <h3 class = "donut-description"><%= @donut.description %></h3>
```

Comment:
take out the spaces around the = for the class = "donut ... so that it's consistent everywhere